schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
   date_of_birth date,
   escape_attempts int,
   neutered boolean,
-  weight_kg decimal
+  weight_kg decimal,
+  species varchar(100) -- update day 2
   );
```

Comment:
- [ ] [REQUIRED] Good job so far! You should `add` a new table with a column called species with a string type like this:

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
I did that already but I forgot to upload the screenshot for it.
Here it is:
<img width="1258" alt="add species column" src="https://user-images.githubusercontent.com/25173553/228563575-741ff4e2-3166-4bd2-841a-8738181d32f1.png">